RetailCoder.VBE/UI/Command/AddTestMethodCommand.cs

```diff
@@ -11,19 +10,18 @@ namespace Rubberduck.UI.Command
     [ComVisible(false)]
     public class AddTestMethodCommand : CommandBase
     {
-        private readonly VBE _vbe;
         private readonly TestExplorerModelBase _model;
+        private readonly NewTestMethodCommand _command;
 
-        public AddTestMethodCommand(VBE vbe, TestExplorerModelBase model)
+        public AddTestMethodCommand(TestExplorerModelBase model, NewTestMethodCommand command)
```

Comment:
okay, this is weird. `AddTestMethodCommand` *is* the old `NewTestMethodCommand` - the implementation of the former static class could simply be carried into this class.